nearcore/src/config.rs

```diff
@@ -948,7 +949,7 @@ pub fn create_testnet_configs_from_seeds(
         if local_ports {
             config.network.addr =
                 format!("127.0.0.1:{}", if i == 0 { first_node_port } else { open_port() });
-            config.rpc.addr = format!("127.0.0.1:{}", open_port());
+            config.rpc.get_or_insert(Default::default()).addr = format!("0.0.0.0:{}", open_port());
```

Comment:
I think it is better to keep 127.0.0.1 here.

```suggestion
            config.rpc.get_or_insert(Default::default()).addr = format!("127.0.0.1:{}", open_port());
```

Reply:
D’oh. Fixed.